feos-core/src/python/state.rs

```diff
@@ -565,7 +565,7 @@ macro_rules! impl_state {
                 self.0.ln_phi().view().to_pyarray(py)
             }
 
-            /// Return logarithmic pure substance fugacity coefficient.
+            /// Return logarithmic fugacity coefficient of a pure liquid reference state.
```

Comment:
maybe change "liquid reference state"  to "...of a pure liquid at mixture pressure and temperature" similar to the rust docstring?